pkg/authorization/generated/clientset/typed/authorization/v1/authorization_client.go

```diff
@@ -40,20 +40,20 @@ func (c *AuthorizationV1Client) ClusterRoleBindings() ClusterRoleBindingInterfac
 	return newClusterRoleBindings(c)
 }
 
-func (c *AuthorizationV1Client) Policies(namespace string) PolicyInterface {
-	return newPolicies(c, namespace)
+func (c *AuthorizationV1Client) Policies() PolicyInterface {
+	return newPolicies(c)
 }
 
-func (c *AuthorizationV1Client) PolicyBindings(namespace string) PolicyBindingInterface {
-	return newPolicyBindings(c, namespace)
+func (c *AuthorizationV1Client) PolicyBindings() PolicyBindingInterface {
+	return newPolicyBindings(c)
 }
 
-func (c *AuthorizationV1Client) Roles(namespace string) RoleInterface {
-	return newRoles(c, namespace)
+func (c *AuthorizationV1Client) Roles() RoleInterface {
```

Comment:
@deads2k @sttts are these supposed to be namespaced or non-namespaced? not sure why this change, it might be bug in my code OR bug in existing code ;-)